symengine/polys/cancel.h

```diff
@@ -0,0 +1,20 @@
+#ifdef SYMENGINE_UINTPOLY_FLINT_H
```

Comment:
You have to change this ifdef to be unique to this file and you have to add #define. Look into how other header files are constructed.

Reply:
Okay @certik